README.md

```diff
@@ -6,7 +6,7 @@
   <p>
 
 [![crates.io](https://img.shields.io/crates/v/actix-web?label=latest)](https://crates.io/crates/actix-web)
-[![Documentation](https://docs.rs/actix-web/badge.svg?version=3.3.2)](https://docs.rs/actix-web/3.3.2)
+[![Documentation](https://docs.rs/actix-web/badge.svg?version=3.3.2)](actix_web)
```

Comment:
The specific versions in the readme links are intentional. Please revert these readme changes as it's easier for me to update when releasing.

The reason is crates.io caches the readme and I want the docs link to reflect the crate version.